src/main/scala/nl/knaw/dans/easy/stage/fileitem/FileItemSettings.scala

```diff
@@ -47,7 +47,7 @@ object FileItemSettings {
             pathInDataset: File,
             format: Option[String],
             size: Option[Long]
-           ) =
+           ): FileItemSettings =
```

Comment:
Why add a type here? Isn't is kind of obvious that `apply` returns a `FileItemSettings`? (the next line instantiates the new object).